### ISSUE TITLE
Add support for icons in dark mode

### ISSUE THREAD

**Issue by hbharding** [elastic/integrations#237]
Some icons like AWS, Kafka, IBM cloud, Github, etc. will not render well if a dark theme is used. In EUI, there is a mechanism for dealing with this, but since we're technically not using EUI icons (rather, we're defining our own custom ones which happen to be identical), we don't get this magic out of the box.

Since the majority of the icons we're using are already defined in EUI, it might be better if we just reference those icons by name instead, and only define custom icons when we need an icon that is not specified in EUI.

So in our manifest:

```
icons:
- src: /img/logo_postgres.svg
  title: logo postgres
  size: 32x32
  type: image/svg+xml
```

could become something much simpler like

```
icons:
- name: logoPostgres
```

I'm not sure why we need to define a title, size, and type. I think one caveat of doing this is that it places a dependency on EUI - meaning any consuming app that wants to grab a package from the registry will need to have EUI setup if they want to see the icon. 

![image](https://user-images.githubusercontent.com/847805/89543581-1ba19d80-d7cf-11ea-810f-3e16ae99e6ea.png)

![image](https://user-images.githubusercontent.com/847805/89543623-278d5f80-d7cf-11ea-8b4b-c7739d47d204.png)

**Comment by mtojek** [comment]
What you're proposing here is a change in manifest. We must handle such issues differently. @ruflin should we transfer it to the spec repo?
/cc @ycombinator

**Comment by ycombinator** [comment]
Yes, lets start using the spec repo for discussions like this. That's exactly one of the reasons we have it now! I will do the transfer.
